tests/test_maintenance_mode.py

```diff
@@ -5,14 +5,14 @@
 
 
 def test_positive_maintenance_mode(setup_sync_plan, ansible_module):
-    """Test foreman-maintain maintenance-mode subcommand
+    """Test satellite-maintain maintenance-mode subcommand
 
     :id: 51d76219-d3cc-43c0-9894-7bcb75c163c3
 
     :setup:
-        1. foreman-maintain should be installed.
+        1. satellite-maintain should be installed.
         2. active sync-plans
-        3. set :manage_crond: true in /etc/foreman-maintain/foreman_maintain.yml'
+        3. set :manage_crond: true in /etc/satellite-maintain/foreman_maintain.yml'
```

Comment:
@tstrych sorry for being nitpicky here, this is a config file, this shouldn't be changed, can you update this?
```suggestion
        3. set :manage_crond: true in /etc/foreman-maintain/foreman_maintain.yml'
```

Reply:
no you are correct here :) I tried to not change paths, I missed this one though